source/_patterns/01-atoms/svgicon/demo/svgicons.twig

```diff
@@ -0,0 +1,49 @@
+{% extends '@atoms/grid/_grid-1-up.twig' %}
+  {% set container = true %}
+  {% block column_1 %}
+
+    <h1>SVG Icons</h1>
+    <p class="lead">Store your SVG files in the SVG subdirectory to have them made available via the <code>_svgicon.twig</code> partial or using a Sass mixin. SVGs are rendered utilizing their filename as a variable.</p>
+
+    <h2>Using the Sass mixin</h2>
+    <p>The Sass mixin is located in the components scss/_icons-generated.scss file which is generated by the Webpack plugin on build. Use the <code>sprite()</code> mixin on your CSS class with the SVG filename as an argument. This class also needs to have a designated height and width.</p> 
+    <pre class="language-css">
+      <code>
+        .svgicon-demo-0 {
+          @include sprite('chevron-right');
+          height: 40px;
+          width: 40px;
+        }
+      </code>
+    </pre>
+      {% for icon in scssIcons %}
+      <div class="icon--{{ icon }}">
+      </div>
+      {% endfor %}
+
+    <h3>Changing colors with the Sass mixin</h3>
+    <p>To allow an SVG to be colorized, add <code>var:color.fill="#000"</code> to all path elements on the raw SVG. This defaults the color to black, but allows an override when using the mixin. <b class="text-danger">Warning:</b> this does involve altering the raw SVG which will need to be be updated with any revisions to the source file.</p>
+    <pre class="language-html">
+      <code>
+        &lt;svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 500 500"&gt;
+          &lt;path var:color.fill="#000" d="M460,463.6c0,19.6-15.9,35.4-35.4,35.4l0,0H69.8c-19.6,0-35.5-15.8-35.6-35.4c0,0,0,0,0-0.1V428 c0-93.4,141.9-141.9,141.9-141.9s8.2-14.5,0-35.5c-29.8-22-33.5-56.4-35.5-141.9C146.8,23.1,206.9,2.3,247.1,2.3 S347.4,23,353.6,108.7c-2,85.5-5.7,119.9-35.5,141.9c-8.2,20.9,0,35.5,0,35.5S460,334.6,460,428V463.6z"/&gt;
+        &lt;/svg&gt;
+      </code>
+    </pre>
+    <pre class="language-css">
+      <code>
+        .svgicon-demo-1 {
+          @include sprite('person', (color: theme-color('primary')));
+          height: 40px;
+          width: 40px;
+        }
+      </code>
+    </pre>
+    <div class="svgicon-demo-1">
+    </div>
+    <h2>Using Twig to render SVG elements</h2>
+    <p>Using the Twig template renders an SVG element directly to the DOM. The template accepts several arguments: <code>svgname</code> accepts the filename of the SVG to render, <code>svgfill</code> accepts a value to set the change the default fill color of the SVG element from black.</p>
+    {% include '@atoms/_svgicon.twig' with {svgname: 'chevron-up'} %}
+    {% include '@atoms/_svgicon.twig' with {svgname: 'chevron-up', svgfill: '#F00'} %}
+
+  {% endblock column_1 %}
```

Comment:
Need to demonstrate the "color svg icon by color name from design system" class concept here.

Reply:
Functionality to use the color name from the design system was added in c4c164c. Further documentation to follow.